content/websites/convex.mdx

```diff
@@ -0,0 +1,34 @@
+---
+name: Convex
+description: The missing half of your React app: Convex is the open-source reactive database for app developers.
```

Comment:
Should be `description: 'The missing half of your React app: Convex is the open-source reactive database for app developers.'`

with single quotes

Reply:
done! let me know if that works.